src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -24,7 +24,7 @@
     "require-dev": {
         "symfony/asset": "~2.8|~3.0",
         "symfony/browser-kit": "~2.8|~3.0",
-        "symfony/console": "~2.8|~3.0",
+        "symfony/console": "~3.2",
```

Comment:
Required for using [`CommandTester::setInputs`](https://github.com/symfony/symfony/pull/20677/files#diff-909255a5d5ed6c36d90b7b8850cee833R147) (see https://travis-ci.org/symfony/symfony/jobs/179580451#L3500)